code/game/objects/items/devices/PDA/apps/general.dm

```diff
@@ -142,7 +142,7 @@ var/global/list/currentevents3 = list("Border patrol around Space America has ti
 	"Renowned mime scientist Free Shrugs has discovered a new element today. He has named it '  ', he also says that it has the properties of '   '.",
 	"Archaeologists have discovered god's final message to his creation today. The message reads, 'bawk'.",
 	"Scientists have discovered a new type of elementary particle today. Our sources say it has a bad atitude, and enjoys the color blue.",
-	"Today, a man was discovered to be living with a 20 year old ghost in his house. When the ghost was questioned who killed him, he responded 'A FAGGOT!'. More at four.",
+	"Today, a man was discovered to be living with a 20 year old ghost in his house. When the ghost was questioned who killed him, he responded 'A Tarjan!'. More at four.",
```

Comment:
It's "Tajaran" but I don't even know how this is supposed to be funny

Reply:
Yeah in this case a more elaborated joke should be made. The joke was the gratuitous use of the homophobic slur which is very funny for whoever wrote it.

OH WAIT IT WAS KANEF. Fucking lol.